release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1562,7 +1556,7 @@ sourceStrategy:
 
 * With this update, {product-title} on {rh-virtualization-first} supports preallocated disks for control plane and worker nodes for installer-provisioned infrastructure. In high-load environments, preallocated disks benefit performance for etcd and other components. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2035334[*BZ#2035334*])
 
-* Previously, when you tried to destroy multiple clusters in parallel, the destroy process failed because of a bug in the `vmware/govmomi` library. The bug caused one of the cluster's tags to be deleted, which resulted in a `404` error when the destroy process could not find the tag. This update ignores tags that are not found and continues the destroy process so that it finishes without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2021041[*BZ#2021041*])
+* Previously, when you tried to delete multiple clusters in parallel, the process failed because of a bug in the `vmware/govmomi` library. The bug caused one of the cluster's tags to be deleted, which resulted in a `404` error when the process to delete could not find the tag. This update ignores tags that are not found and continues to delete so that it finishes without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2021041[*BZ#2021041*])
```

Comment:
```suggestion
* Previously, when you tried to delete multiple clusters in parallel, the process failed because of a bug in the `vmware/govmomi` library. The bug caused  a cluster tag to be deleted, which resulted in a `404` error when the process to delete could not find the tag. This update ignores tags that are not found and continues to delete so that it finishes without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2021041[*BZ#2021041*])
```